README.md

```diff
@@ -3,11 +3,19 @@
 ## Quick Start
 
 1. Clone this repo to your local environment.
+1. Make sure you have Ruby installed by running `ruby -v`
+1. Run `bundle install` to install required gems. If you don't have Bundler installed, you can install it with `gem install bundler`
 
-2. Run `bundle install` to install required gems. If you don't have Bundler installed, you can install it with `gem install bundler`
+## Command Line Usage
 
-3. To start your server, you can run the `rackup` command. You can also start it by running `bin/start-localhost-server.sh`, but be sure to make that file executable. `chmod -x bin/start-localhost-server.sh`
+Run `./bin.convert` passing in the roman numeral you would like to convert as an argument, like so:
 
-4. Once your server is running, you can curl localhost:8000 (or whichever port your server is using), or access in the browser. Specify the /convert route and pass in the numeral you want converted, like so: `curl localhost:8000/convert?roman=XVIII`
+    `./bin/convert <numeral>`
```

Comment:
In order to run this without specifying Ruby (`ruby ./bin/convert`) I feel like the user needs to make this fine executable. Am I right about that?

Reply:
Good question. The file is executable as committed to the repo. You can tell this if you `chmod` a file in a git repo, it shows up as `modified`. So I think we can leave this as is.